tests/TestableIO.System.IO.Abstractions.Wrappers.Tests/ApiParityTests.cs

```diff
@@ -24,6 +24,13 @@ public void FileInfo() =>
                 typeof(System.IO.Abstractions.FileInfoBase)
             );
 
+        [Test]
```

Comment:
Can you also add a parity test fo the `IFileVersionInfoFactory`?

Reply:
I'm confused. There are no parity tests for the factories. To what should I compare the `IFileVersionInfoFactory`?

Reply:
Sorry, my bad. I have parity tests for factories in [Testably.Abstractions](https://github.com/Testably/Testably.Abstractions/blob/main/Tests/Testably.Abstractions.Parity.Tests/ParityTests.cs) and got confused :-)

Reply:
All good :) Then I resolve this one.